### PR TITLE
Mock data

### PR DESCRIPTION
**Pull request by gcoleman799**
This PR adds a folder containing the sample data and changes the location where the JSON is being fetched from.

### CODE REVIEW

File: WebView/app/src/main/assets/main.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function sendAndroidMessage() {
 }
 
 function getData() {
-	fetch("https://gcoleman799.github.io/Asset-Loader/weather.json").then(function(resp) {
+	fetch("https://github.com/android/views-widgets-samples/tree/webview/WebView/sampleData/weather.json").then(function(resp) {
```
Comment (nic0lette) [inline WebView/app/src/main/assets/main.js:26]: Should this be `raw.githubusercontent.com` rather than `github.com` ?

Comment (gcoleman799) [reply WebView/app/src/main/assets/main.js:26]: Yes, I think so. I am going to remove the change to where the data is being fetched from. This way we can test that the URL is correct before we make the change.

File: WebView/sampleData/weather.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "newYork":{
+    "currentTemp":"75°",
+    "highTemp":"83°",
+    "lowTemp":"64°",
+    "chancePrecip":"90%",
+    "humidity":"23%",
+    "description":"Rainy",
+    "icon":"https://gcoleman799.github.io/res/drawable/rain.png"
```
Comment (nfischer) [inline WebView/sampleData/weather.json:9]: Can we move the icon URL out of the json? I feel like the app should have logic to choose the icon based on the description.

File: WebView/sampleData/weather.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "newYork":{
+    "currentTemp":"75°",
+    "highTemp":"83°",
+    "lowTemp":"64°",
+    "chancePrecip":"90%",
+    "humidity":"23%",
```
Comment (nfischer) [inline WebView/sampleData/weather.json:7]: Can we change each of these to Numbers and let the app add the appropriate units? Also, temperature should clarify which units (C or F).

Comment (gcoleman799) [reply WebView/sampleData/weather.json]: I will make these changes to the JSON here and then I will create a follow up PR changing where the app is fetching data from along with the adjustments needed on the app side

**Pull request merged**